pkgs/development/libraries/gstreamer/rs/default.nix

```diff
@@ -165,7 +166,16 @@ stdenv.mkDerivation rec {
     pkg-config
     rustc
     cargo
-    cargo-c
+    (cargo-c.overrideAttrs (oldAttrs: {
```

Comment:
Patching Inline like this has a chance to miss other usages of the package which wouldn't be ideal.

Reply:
Like other usages within nixpkgs or other usages within the same derivation?

If you have a suggestion, I don't particularly mind changing how this is done if the result is the same

Reply:
> other usages within the same derivation?

that one mostly.

I think this could either be moved into a let in where you probably want to rename it to cargo-c' or in callPackage. I think we would want to do the first here.

Reply:
I pushed an update to move `cargo-c'` into the `let ... in` in the derivation